### PR TITLE
(feat) O3-3955: Add question info text field to the interactive builder

### PR DESCRIPTION
**Pull request by andie23**
# Changes
1. Added new type in Question interface
2. Updated Add Question Editor and Edit Question Editor to include question info

### CODE REVIEW

File: src/const.ts
Comment (NethmiRodrigo) [inline src/const.ts:1]: I think this is something we can move into the `config-schema.ts`

File: src/types.ts [hunk 1]
```diff
@@ -117,6 +117,7 @@ export interface Question {
   id: string;
   label: string;
   type: string;
+  questionInfo?: string;
```
Comment (NethmiRodrigo) [inline src/types.ts:120]: Shall we update the `Schema` interface as well to include `questionInfo`?

**Review by NethmiRodrigo — COMMENTED** [review]
Great work @andie23!! Just some minor nitpicks. Also, could you please update your PR description to follow the template? It helps a ton when reviewing :)

**Comment by NethmiRodrigo** [comment]
Closing since its inactive, please feel free to re-open this if you begin working on this again. Thanks!

**Comment by harshthakkr** [comment]
@NethmiRodrigo If you’re okay with it, I can create a new PR with the same changes so we can move forward.

**Pull request closed without merging**